api-docs/cloud-dedicated/ref.yml

```diff
@@ -1442,14 +1442,29 @@ paths:
 
         Use this endpoint to send data in [line protocol](/influxdb/cloud-dedicated/reference/syntax/line-protocol/) format to InfluxDB.
 
-        InfluxDB does the following when you send a write request:
+        InfluxDB Cloud Dedicated does the following when you send a write request:
 
-        1. Validates the request
-        2. If successful, attempts to [ingest the data](/influxdb/cloud-dedicated/reference/internals/durability/#data-ingest); _error_ otherwise.
-        3.  If successful, responds with _success_ (HTTP `204` status code), acknowledging that the data is written and queryable; _error_ otherwise.
+        1. Validates the request.
+        2. If successful, attempts to [ingest data](/influxdb/cloud-dedicated/reference/internals/durability/#data-ingest) from the request body; otherwise, responds with an [error status](/influxdb/cloud-dedicated/write-data/troubleshoot/#review-http-status-codes).
+        3. Ingests or rejects data in the batch and returns one of the following HTTP status codes:
 
-        To ensure that InfluxDB Cloud handles writes in the order you request them,
-        wait for a success response (HTTP `2xx` status code) before you send the next request.
+           - `204 No Content`: all data in the batch is ingested
+           - `201 Created` (_If the cluster is configured to allow **partial writes**_): some points in the batch are ingested and queryable, and some points are rejected
```

Comment:
As noted in Slack, I think this approach to documenting optional behavior is good. I think we just need to link to somewhere that provides information on how users can have their cluster configured for partial writes.